src/assignment/views/AssignmentResponseEdit/AssignmentResponseEdit.tsx

```diff
@@ -373,6 +373,7 @@ const AssignmentResponseEdit: FunctionComponent<
 				}
 				return (
 					<AssignmentResponsePupilCollectionTab
+						pastDeadline={pastDeadline}
```

Comment:
Avoid unneeded prop after `pastDeadline` is split up.

Reply:
AssignmentResponsePupilCollectionTab has no access to assignment, so we still need to pass the pastDeadline boolean
otherwise we need to pass the assignment object and have the component caluclate it it self with an extra hook call.